### ISSUE TITLE
Resilience 4j CB health report giving different answers each time

### ISSUE THREAD

**Issue by u742936** [resilience4j/resilience4j#1481]
Resilience4j version: 1.4.0 

Java version: 1.8 

Hi, I’m using spring boot annotations for Resilience4j CB along with retry

in my local everything works well, CB going to OPEN after threshold is met. And actuator/health shows the exact number of calls made

While in my SIT environment when same code is deployed, I see that CB is not going to OPEN, when I run health/report each time i run it gives different number of calls in report even when no calls are made.  
until it reaches minimum number of calls and later always giving me same health report 

say failurethreaholdrate is 50, minimumnumberofcalls is 10 and retry is 3 
That means calls will be always increments of 3. 3,6,9 after reaching 10, 11th call should go to open. 

1st call:
_________
after my 1st failed call, checking health report should show 3 failed calls every time, but is shows 0 once and next time 3 and next time 0 next time 3, it keeps fluctuating

2nd call:
__________
after making next call, so total we made 6 calls,
But health report shows 3 once and next time 6 and next time 3 next time 6, it keeps fluctuating

3rd call:
__________
and for next call health report is fluctuating every time, once it shows 6 another time 9, next run it shows 6 and next time 9

4th call, 5th call and all calls:
________________________________
After reaching 9 failed calls, any number of attempts I make, always the failed calls is 9 it is not opening the circuit

and it is happening in SIT env

I feel like the CB Got corrupted even if the restart the server or even after deploying my previous versions which actually worked, all started behaving crazy like stated above

Can you help me how to fix CB in corrupted cases like this 

Thanks for raising a Resilience4j issue.
Please replace this text with a brief description of your problem along with the versions you are using.
If possible, please also consider putting together a complete JUnit test that reproduces the issue.

**Comment by RobWin** [comment]
Hi,
I've never heard of such behavior. Without seeing your code it's difficult to help.
Maybe you should update to the latest version as well

**Comment by u742936** [comment]
In SIT env where the code is deployed in 2 servers, looks like the Health report is Showing for each individual server separately every time I run the report from common F5 url.

**Comment by RobWin** [comment]
Yes, every application instance has it's own health endpoint and it's own state.
Application instances don't have a state which is distributed over all instances.
